packages/wcg-crypto/references.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference path="../../references.d.ts" />
```

Comment:
Ah I guess this would be the ideal place for the triple-slash directive I mentioned. Maybe no need to add it to each individual file? Depends how the users end up importing the package, because ultimately it's gonna be a polyfill 🤔